### PR TITLE
Merge release/vm poc

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "yagna"
-version = "0.3.0"
+version = "0.3.3-alpha.4"
```
Comment (tworec) [inline Cargo.toml:3]:
```suggestion
version = "0.3.3-alpha.5"
```

Comment (mfranciszkiewicz) [reply Cargo.toml:3]: @tworec I'd suggest `0.4.0` since ExeUnit uses a completely different communication api with the vm runtime

Comment (tworec) [reply Cargo.toml:3]: is it back compatible? I need some sync with you on that

Comment (tworec) [reply Cargo.toml:3]: but this seems legit afai understand

Comment (mfranciszkiewicz) [reply Cargo.toml:3]: @tworec on a second thought, `ya-exe-unit` and `ya-provider` are versioned separately, so I'm leaving this change to other reviewers

**Review by prekucki — DISMISSED** [review]

**Comment by mfranciszkiewicz** [comment]
Closing in favour of #543

**Pull request closed without merging**